runtime/queries/poe_filter/highlights.scm

```diff
@@ -36,6 +36,6 @@
 "\"" @punctuation.delimiter
 
 ; conceal unnecessary quotes
-("\"" @conceal
-  (#not-has-parent? @conceal string file)
+("\"" @_conceal
+  (#not-has-parent? @_conceal string file)
   (#set! conceal ""))
```

Comment:
Just noting: This will still be captured as `@_conceal.poe_filter`, so we can still give it highlight groups

Reply:
But we don't want to, in this case, since the highlight is already applied in the previous pattern?

Reply:
:thinking:

Reply:
I'd go for duplicating the highlight captures. At least it's consistent highlighting with or without conceal

Reply:
> Oh, it's true, the @_conceal capture shows up in Inspect.

But so does `@conceal`? (I'm fine with keeping the highlight, though.)